pages/apis/rest_api/analytics/runs.md

```diff
@@ -45,3 +49,7 @@ curl "https://api.buildkite.com/v2/analytics/organizations/{org.slug}/suites/{su
 Required scope: `read_suites`
 
 Success response: `200 OK`
+
+Runs are created with a `state` of `running` and proceed to `finished` when all uploads have been processed. The run may return to `running` if additional results are uploaded.
```

Comment:
```suggestion
Runs are created with a `state` of `running` and proceed to `finished` when all uploads have been processed. The run may return to `running` if additional results are uploaded. If a run receives no results within a reasonable time period its `state` will proceed to `stale`.
```

Reply:
FYI our default scope on runs excludes runs that are `stale`, so I think the behaviour might be that the API will returning the `running` run if requested, but once the run becomes `stale` the same API request will probably return a not found or similar?

Reply:
This would happen now with the API as well so I'm not suggesting a change in behaviour, I was just wondering maybe if we shouldn't mention the `stale` state at all because it wouldn't ever be returned to the customer in the API

Reply:
I don't think this is correct. `result` can become `stale`, but `state` can only be `running` or `finished`.

Reply:
Oh, I totally thought `stale` was a `state` and not a `result`, but yeah that's not what the code says!